src/garage/torch/modules/cnn_module.py

```diff
@@ -0,0 +1,217 @@
+"""CNN Module."""
+import copy
+
+from torch import nn
+
+from garage.torch import flatten_to_single_vector, NonLinearity
+
+
+# pylint: disable=unused-argument
+class CNNModule(nn.Module):
+    """Convolutional neural network (CNN) model in pytorch.
+
+    Args:
+        input_var (pytorch.tensor): Input tensor of the model.
+            Based on 'NCHW' data format: [batch_size, channel, height, width].
+        kernel_sizes (tuple[int]): Dimension of the conv filters.
```

Comment:
It will be better for me if the `kernel_sizes` is consistent to the [TF tree](https://github.com/rlworkgroup/garage/blob/e2c890d404eb8c40e90b480d2cb931cb21dbea74/src/garage/tf/models/cnn.py#L24).